services/dy-jupyter-extensions/neuron/requirements/base.txt

```diff
@@ -0,0 +1,88 @@
+aiofiles==0.4.0
```

Comment:
should there not be a base.in as well?

Reply:
I wanted to freeze the python installation to the version that is there from the base.